lib/measured/conversion.rb

```diff
@@ -48,6 +48,9 @@ def conversion_table
 
   def unit_for(name)
     unit = @units.find { |unit| unit.names_include?(name, case_sensitive: @case_sensitive) }
-    unit or raise Measured::UnitError, "Cannot find unit for #{name}"
+  end
+
+  def unit_for!(name)
+    unit_for(name) or raise Measured::UnitError, "Cannot find unit for #{name}"
```

Comment:
Avoid `or` entirely. It reads nicely, but the subtle order of operations difference is confusing and causes nasty bugs (ie. lower precedence than `=` for example). It's pretty [well accepted ruby style](https://github.com/bbatsov/ruby-style-guide#no-and-or-or) to just not use them.